dump/dump_collect.cpp

```diff
@@ -122,6 +122,47 @@ void collectDumpFromSBE(struct pdbg_target* chip,
     {
         openpower::phal::sbe::getDump(chip, type, clockState, collectFastArray,
                                       dataPtr.getPtr(), &len);
+        // if dump operation is success, check if there is any FFDC data if
+        // found create pel for it
+        if (isOcmb)
+        {
+            try
+            {
+                openpower::phal::sbeError_t error =
+                    openpower::phal::sbe::capturePOZFFDC(chip);
+                if (error.errType() ==
+                    openpower::phal::exception::SBE_CMD_FAILED)
+                {
+                    openpower::dump::pel::FFDCData pelAdditionalData;
+                    uint32_t cmd = SBE::SBEFIFO_CMD_CLASS_DUMP |
+                                   SBE::SBEFIFO_CMD_GET_DUMP;
+
+                    pelAdditionalData.emplace_back(
+                        "SRC6", std::to_string((chipPos << 16) | cmd));
```

Comment:
For ocmb we use FAPI position in most of the places. Ln. 100 **auto chipPos = pdbg_target_index(chip);**
Are we supposed to capture index or FAPI position here? Because index could repeat. Right?
ocmb0 {
index = <0x0d>;
ATTR_FAPI_POS = <29>;
ATTR_PHYS_DEV_PATH = "physical:sys-0/node-0/ocmb_chip-51